openmetadata-spec/src/main/resources/json/schema/entity/services/connections/database/athenaConnection.json

```diff
@@ -47,6 +48,11 @@
       "description": "Athena workgroup.",
       "type": "string"
     },
+    "database": {
+      "title": "Database",
+      "description": "Database of the data source. This is optional parameter, if you would like to restrict the metadata reading to a single database. When left blank, OpenMetadata Ingestion attempts to scan all the databases.",
```

Comment:
we might need to change this description, we will have to make it clear that this is just for display and that we will not actually ingest the database.